### PR TITLE
Add figure <number> to text/caption _index.md

### PR DESCRIPTION
**Pull request by chrismetz09**
Change file: ../docs/contribute/_index.md

Method described in [How to use captions](https://kubernetes.io/docs/contribute/style/diagram-guide/#how-to-use-captions)

Contributes to common method for referencing and labeling figures in docs.

### CODE REVIEW

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-31654--kubernetes-io-main-staging.netlify.app/docs/contribute/#getting-started

File: content/en/docs/contribute/_index.md [hunk 1]
```diff
@@ -95,17 +95,17 @@ class A,B,C,D,E,F,G,H,M,Q,N,O,P,V grey
 class S,T,U spacewhite
 class first,second,third white
 {{</ mermaid >}}
-***Figure - Getting started for a new contributor***
+Figure 1. Getting started for a new contributor.
```
Comment (kbhawkey) [inline content/en/docs/contribute/_index.md:98]: Does Mermaid provide a way to label a figure?

**Comment by kbhawkey** [comment]
Would you rebase to a single commit? Thanks!
/lgtm

File: content/en/docs/contribute/_index.md (hunk 1 above)
Comment (chrismetz09) [reply content/en/docs/contribute/_index.md:98]: Has been discussed per [add title to graph issue](https://github.com/mermaid-js/mermaid/issues/1433) but nothing implemented at this time.

**Comment by chrismetz09** [comment]
@kbhawkey, tx for the review. 

https://deploy-preview-31654--kubernetes-io-main-staging.netlify.app/docs/contribute/

**Pull request merged**